src/backend/tests/mocked/reimbursement-requests.test.ts

```diff
@@ -439,13 +439,12 @@ describe('Reimbursement Requests', () => {
 
     test('Delete Reimbursement Request fails when deleter is not the creator', async () => {
       vi.spyOn(prisma.reimbursement_Request, 'findUnique').mockResolvedValue(GiveMeMyMoney);
-
       await expect(() =>
         ReimbursementRequestService.deleteReimbursementRequest(GiveMeMyMoney.reimbursementRequestId, superman)
```

Comment:
also separate the tests: case 1: when the user is not creator. case 2: when the user is not finance lead+. case 3: when user is finance but not leadership